tiny_cnn/core/conv_params.h

```diff
@@ -90,6 +97,10 @@ struct conv_params {
     padding pad_type;
     size_t w_stride;
     size_t h_stride;
+    // deconvolution parameters
+    index3d<cnn_size_t> in_;
+    index3d<cnn_size_t> out_;
+    index3d<cnn_size_t> out_unpadded_;
```

Comment:
@edgarriba And here, parameter for convolution and deconvolution could be in the same structure.